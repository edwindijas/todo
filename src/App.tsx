function App (): JSX.Element {
  return (
    <div className='w-full h-full flex items-center justify-center' >
      Hello World
    </div>
  )
}

export default App
