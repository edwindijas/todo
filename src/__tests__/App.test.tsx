import { type RenderResult, render } from '@testing-library/react'
import App from '../App.tsx'

const renderComponent = (): RenderResult => render(<App />)

it('correctly renders', () => {
  const { container } = renderComponent()
  expect(container).toMatchSnapshot()
})
