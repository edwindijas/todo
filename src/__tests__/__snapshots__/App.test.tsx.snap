// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly renders 1`] = `
<div>
  <div
    class="w-full h-full flex items-center justify-center"
  >
    Hello World
  </div>
</div>
`;
